### DISCUSSION TITLE
Translation for Chinese (Simplified)

### DISCUSSION THREAD

**Discussion by CasperYC** [rgriebl/brickstore#566] [Ideas]
Hi,

Just found out about this wonderful offline tool for Lego.
If you need a volunteer for translation into Simplified Chinese,
I am more than happy to do that.

Just drop me a message!

**Comment by rgriebl** [comment]
Hi,
Thanks for volunteering! Please have a look at this conversation regarding the Swedish translation #551 to get started.
I've also attached an empty zh_CN (simplified Chinese) translation right here:

[brickstore_zh_CN.zip](https://github.com/rgriebl/brickstore/files/9084997/brickstore_zh_CN.zip)

**Reply by CasperYC** [reply]
Great. I have done several translations on seperate occasions with Qt, but zero experience in compiliing from source code.
I will have a go and let you know how I get on!

**Reply by CasperYC** [reply]
Hi there,

Could you please re-create the template for zh_CN as the one above is completely empty and Qt couldn't open it.

**Reply by rgriebl** [reply]
You don't need to compile from source! It's just more convenient to HAVE the source code 
AVAILABLE, because Linguist will show you a live preview of most dialogs while you are translating this way. (and even if the translated string is not in a preview-able dialog, you still see the source-code, which might help with getting a better idea what context the string occurs in).

And sorry about the empty TS file: here's the correct one
[brickstore_zh_CN.zip](https://github.com/rgriebl/brickstore/files/9092365/brickstore_zh_CN.zip)

**Reply by CasperYC** [reply]
![Capture](https://user-images.githubusercontent.com/54921200/178498526-7d79d818-dc38-4ec7-8ee0-5276b87c88b6.PNG)


Great!
I used the English version and started translating it this afternoon already.

It's now work in progress...

It does have a lot of string to translate!!!
